internal/platform/stat_windows.go

```diff
@@ -63,6 +60,18 @@ func statFile(f fs.File, st *Stat_t) (err error) {
 	return defaultStatFile(f, st)
 }
 
+// inoFromFileInfo uses stat to get the inode information of the file.
+func inoFromFileInfo(f readdirFile, t fs.FileInfo) (ino uint64, err error) {
+	if pf, ok := f.(PathFile); ok {
+		var st Stat_t
+		inoPath := path.Clean(path.Join(pf.Path(), t.Name()))
+		if err = lstat(inoPath, &st); err == nil {
```

Comment:
this is where we use our special interface method in order to build up the path to the directory entry. lstat is the right choice vs stat for dirents, as for example, links aren't followed.